### ISSUE TITLE
LaTeX inline preview in org-mode fails to render on develop

### ISSUE THREAD

**Issue by trentfridey** [syl20bnr/spacemacs#13988]
#### Description :octocat:
LaTeX inline preview in org-mode fails to render on develop
My org mode version is 9.3 (release 9.3 @ elpa/27.1/develop/org-plus-contrib-20200921)

#### Reproduction guide :beetle:
- Start Emacs 
- Open a new .org file
- Add a title
- Type `$a^2 + b^2 = c^2$` in the body
- Place point on expression
- Trigger LaTeX inline preview with [, T x] 

*Observed behaviour:* :eyes: :broken_heart:
Message reported: "Symbol's function definition is void: org-collect-keywords"
LaTeX inline preview fails to render

*Expected behaviour:* :heart: :smile:
Org-mode produces a preview image of the LaTeX fragment at point and overlays it over the source code.

#### System Info :computer:
- OS: gnu/linux
- Emacs: 27.1
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. 8b888c6a0)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(emacs-lisp helm latex python multiple-cursors org treemacs)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG CAIRO SOUND GPM DBUS GSETTINGS GLIB NOTIFY INOTIFY ACL LIBSELINUX GNUTLS LIBXML2 FREETYPE HARFBUZZ M17N_FLT LIBOTF ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 XDBE XIM MODULES THREADS XWIDGETS LIBSYSTEMD JSON PDUMPER LCMS2 GMP

**Comment by emacs18** [comment]
This works fine for me.  I use ubuntu 20.04, emacs 27 I built on Sep 20, org mode also from Sep 20, spacemacs version
8b888c6a0.

**Comment by trentfridey** [comment]
So I managed to get the latex preview working, but I am not sure why.

I blew away my Emacs installation, as well as any config files for emacs / spacemacs, reinstalled Emacs, as well as spacemacs. I rebuilt Emacs 27, cloned spacemacs into a fresh .emacs.d/, and booted Emacs with spacemacs on the development branch (HEAD at 5812d52e6).

Following the reproduction steps left me with the same message: "Symbol's function definition is void: org-collect-keywords"

Then I checked out 8b888c6, opened Emacs with the `--debug-init` flag, and tried the reproduction steps again. It worked!  🎉
So I thought is was a regression, but I checked out 5812d52e6, and tried to reproduce the error behavior again, but it is rendering inline previews just fine! 🤷

**Comment by duianto** [comment]
Could it have been fixed in org 9.4? It was released recently.

I got two Package Installation dialog boxes about missing files, when I pressed `, T x`

![latex_2020-09-30_10-36-57](https://user-images.githubusercontent.com/13420573/94663142-9daece00-0309-11eb-97be-b90c3b67a66a.png)

![dvipng_2020-09-30_10-37-37](https://user-images.githubusercontent.com/13420573/94663151-a0112800-0309-11eb-8708-a8a17abfeb7c.png)

But then it seems to have worked:

![emacs_2020-09-30_10-38-37](https://user-images.githubusercontent.com/13420573/94662999-71934d00-0309-11eb-8af1-c5e2f870259a.png)

With updated packages on the latest develop branch: ce7c2f05d
<details><summary>
Windows 10, System Info (Click to expand)
</summary>
<pre>
#### System Info :computer:
- OS: windows-nt
- Emacs: 27.1
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. ce7c2f05d)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(autohotkey command-log emacs-lisp git go haskell helm
            (javascript :variables javascript-backend 'lsp)
            (languagetool :variables langtool-language-tool-jar "C:\\Users\\username\\Applications\\LanguageTool-5.0\\languagetool-commandline.jar")
            lsp markdown multiple-cursors org
            (python :variables python-backend 'lsp python-lsp-server 'mspyls)
            shell spell-checking version-control tabs treemacs)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG SOUND NOTIFY W32NOTIFY ACL GNUTLS LIBXML2 HARFBUZZ ZLIB TOOLKIT_SCROLL_BARS MODULES THREADS JSON PDUMPER LCMS2 GMP

</pre>
</details>
